src/layout/SideBar.tsx

```diff
@@ -0,0 +1,50 @@
+import { TextField, FormGroup, Box, Typography } from '@mui/material';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { FilterList } from '@mui/icons-material';
+import FilterCheckbox from '../components/FilterCheckbox';
+
+// TODO: Needed to make fixed nav
+/* import { makeStyles } from '@material-ui/core/styles';
+ const useStyles = makeStyles((theme) => ({
+   toolbar: theme.mixins.toolbar,
+ }));
+*/
+
+function SideBar() {
+  const theme = createTheme({
+    palette: {
+      primary: {
+        main: '#E1B15D', // brown
+      },
+      secondary: {
+        main: '#FFFFFF', // white
```

Comment:
can you have 1 place for all our primary/secondary/tertiary colours ? and inherit/reference it...that way we dont have to make chagnes in multiple places

Reply:
yea, just made a fix for it